python/core/auto_generated/geometry/qgswkbtypes.sip.in

```diff
@@ -128,6 +130,22 @@ Returns the multi type for a WKB type. For example, for Polygon WKB types the mu
 %End
 
 
+    static Type promoteNonPointTypesToMulti( Type type ) /HoldGIL/;
+%Docstring
+Promotes a WKB geometry type to its multi-type equivalent, with the exception of point geometry types.
+
+Specifically, this method should be used to determine the most-permissive possible resultant WKB type which can result
+from subtracting parts of a geometry. A single-point geometry type can never become a multi-point geometry type as
+a result of a subtraction, but a single-line or single-polygon geometry CAN become a multipart geometry as a result of subtracting
+portions of the geometry.
+
+.. seealso:: :py:func:`multiType`
+
+.. seealso:: :py:func:`singleType`
+
+.. versionadded:: 3.24
```

Comment:
since it's backported, it should be 3.22.2, right?

Reply:
argh sorry, missed that comment!

Ideally yes, but I also don't think it's worth the effort of manually changing the backport accordingly...